commons/src/test/java/io/hyscale/commons/utils/LogProcessorTests.java

```diff
@@ -16,31 +16,29 @@
 package io.hyscale.commons.utils;
 
 import io.hyscale.commons.exception.HyscaleException;
-import io.hyscale.commons.handler.SampleLogHandler;
+import io.hyscale.commons.handler.TestTailLogHandler;
 import org.apache.commons.io.FileUtils;
 import org.junit.jupiter.api.*;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
-
+import org.junit.jupiter.params.provider.*;
+import org.junit.jupiter.api.Nested;
 import java.io.*;
 import java.nio.charset.Charset;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class LogProcessorTests {
-    private static String logFileContent = "logger running" + "\n" + "logger Running";
-    private LogProcessor logProcessor = new LogProcessor();
-    private static String logFilePath = "/tmp/logs/logs.txt";
-    private static String encoding = "UTF-8";
+    private static final String LOG_FILE_PATH = "/tmp/logs/logs.txt";
+    private static final LogProcessor logProcessor = new LogProcessor();
+    private static final String ENCODING = "UTF-8";
+    private static String logFileContent = "logger running" + "\n" + "logger running";
     private static File file;
-    private SampleLogHandler sampleLogHandler = new SampleLogHandler();
-
+    private static ByteArrayOutputStream os = new ByteArrayOutputStream();
```

Comment:
Why is this global and shared across tests ?

Reply:
Since i am sending as argument in the static method in the global class.